src/sub/tests/unit/test_exceptions.py

```diff
@@ -73,7 +95,7 @@ def test_server_error():
     assert ex2.to_dict() == dict(message=message, some="payload")
 
 
-def test_SecretStringMissingError():
+def test_secret_string_missing_error():
```

Comment:
this is unrelated to this change; also `SecretStringMissingError` is a specific name for the error class to be tested, therefore it is inappropriate to lowercase and separate by `_`.

Reply:
done